Fix 4448

GlennTatum opened the pull request:
## Summary
Fixes #4448 

### Time to review: 10 Minutes

## Changes proposed
> What was added, updated, or removed in this PR.
Forecasted grants now show forecasted metadata under `OpportunityStatusWidget`

## Context for reviewers
> Testing instructions, background context, more in-depth details of the implementation, and anything else you'd like to call out or ask reviewers. Explain how the changes were verified.

If a `summary.close_date_description` is not present the description will be set as "Not available" the i18n translations were updated for this specific case.

## Additional information
> Screenshots, GIF demos, code examples or output to help show the changes working as expected.

![image](https://github.com/user-attachments/assets/79c9d8f2-6834-47b9-ab28-7683ddf49cc4)

GlennTatum commented:
![image](https://github.com/user-attachments/assets/59e5013b-c56a-44d9-a26e-5e95bafebf0c)

GlennTatum commented:
> Thanks for putting together that table [@chrisw-nava](https://github.com/chrisw-nava)! I agree that we don't need post_date in the status widget.
> 
> [@GlennTatum](https://github.com/GlennTatum), the screenshot looks awesome! I'd say we can remove "Last Updated Date" because it is above the "Funding Opportunity" heading on the left.

Finished up a request from the UI team and ran prettier to format the code. 

Let me know if there needs to be any additional changes for this to go through CI successfully. @btabaska

btabaska commented:
This looks like it has issues with formatting:
Checking formatting...
[warn] src/styles/_uswds-theme-custom-styles.scss
[warn] Code style issues found in the above file. Run Prettier with --write to fix.

GlennTatum commented:
```
$ npm run format-check

> frontend@0.1.0 format-check
> prettier --check '**/*.{js,json,md,ts,tsx,scss,yaml,yml}'

Checking formatting...
All matched files use Prettier code style!
```

Tested locally with `npm run format-check`

GlennTatum commented:
The branch has been synced to the most recent commit.
```
npm run format-check

> frontend@0.1.0 format-check
> prettier --check '**/*.{js,json,md,ts,tsx,scss,yaml,yml}'

Checking formatting...
All matched files use Prettier code style!
```

However, I have changed the scss file to match _exactly_ with the current revision, although npm run format-check may be succeeding locally, I have changed it to a warning state to see if the CI pipeline may have different opinions.

GlennTatum commented:
On vscode I am using `11.0.0` of the Prettier extension

However, npm is using:
```
npm exec prettier --version
10.7.0
```

The primary issue is occuring here is `url` is below mask-image:
```scss
      span:after {
        mask-image:
          url("/uswds/img/usa-icons/expand_less.svg"),
          linear-gradient(transparent, transparent);
        background-color: white;
      }
    }
    button[aria-expanded="false"] {
      span:after {
        mask-image:
          url("/uswds/img/usa-icons/expand_more.svg"),
          linear-gradient(transparent, transparent);
      }
    }
```
Where mask-image: 
Instead of having  the `url `to the mask-image below:
```diff
    mask-image:
-      url("/uswds/img/usa-icons/expand_more.svg"),
      linear-gradient(transparent, transparent);
```
Should be theoretically inline instead as on the same line as `mask-image`:
```diff
+ mask-image: url("/uswds/img/usa-icons/expand_more.svg"),
```

The version discrepancy between the vscode `Prettier` extension versions is what is most likely causing the issue in CI.

Although community extension versions of vscode are not able to utilize the `extensions.allowed` within the `.vscode/` settings configuration (exclusive to enterprise uses), it would be nice if there was a method of being able to have contributors from being able to accidentally format files using the built-in extension over the current prettier executable installed by npm.

GlennTatum commented:
Update: The Prettier version discrepancy was the problem

btabaska reviewed (approved):
LGTM!
